### PR TITLE
Add serviceable attribute to projects.

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
Updated.

File: src/Microsoft.Framework.CodeGeneration.Common/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyMetadata("Serviceable", "True")]
```
Comment (Eilon) [inline src/Microsoft.Framework.CodeGeneration.Common/Properties/AssemblyInfo.cs]: Looks like this is a shared source project.

**Comment by Eilon** [comment]
@PradeepKadubandi in case you want to take a look.

File: src/Microsoft.Framework.CodeGeneration.Common/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (NTaylorMullen) [inline src/Microsoft.Framework.CodeGeneration.Common/Properties/AssemblyInfo.cs]: Yup, updated script to account for *.cs shared projects.

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**